src/utilcode/winwrappers.cpp

```diff
@@ -0,0 +1,996 @@
+// Licensed to the .NET Foundation under one or more agreements. 
+// The .NET Foundation licenses this file to you under the MIT license. 
+// See the LICENSE file in the project root for more information. 
+
+#include "stdafx.h"
+#include "windows.h"
+#include "winwrappers.h"
+#include "sstring.h"
+#include "ex.h"
+
+typedef InlineSString<2*MAX_PATH> Win32StackSString;  //2*MAX_PATH - is tunable stack allocated buffer size
```

Comment:
We should move this definition to sstring.h, which contains a similar PathString definition. Also, I would suggest we name this more appropriately - something like LongPathString.